setup.cfg

```diff
@@ -5,24 +5,32 @@ description-file =
     README.rst
 author = Redfish dev team
 author-email = python-redfish@mondorescue.org
-home-page = http://github.com/bcornec/python-redfish
+home-page = https://github.com/uggla/python-redfish/tree/devel
```

Comment:
I've already added you as a contributor to my tree. That should ease with bug reports. I also agree for the PR which seems an interesting way to progress. All your points are valid ones, so we go for that. Let me discuss also with Vincent to check before adding him.

It's still not clear wrt to Devananda and his repo. But we can work on that in //.